### ISSUE TITLE
Elasticsearch exporter periodical bulk flushing error

### ISSUE THREAD

**Issue by edward-sh** [camunda-cloud/zeebe#5404]
**Describe the bug**
Getting errors during flushing bulk in Elasticsearch exporter.  We are using single node of Zeebe Broker in AWS K8S cluster with external Elasticsearch three-nodes cluster. What could be the possible reason of such behavior? Thanks in advance for your help!

<details><summary>Full Stacktrace</summary>
 <p>

```
2020-09-22 14:47:32.405 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] WARN  io.zeebe.broker.exporter.elasticsearch - Failed to flush 6 item(s) of bulk request [type: mapper_parsing_exception, reason: object mapping for [value.variables.exception] tried to parse field [exception] as object, but found a concrete value]
2020-09-22 14:47:32.405 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] ERROR io.zeebe.broker.exporter.elasticsearch - Unexpected exception occurred on periodically flushing bulk, will retry later.
io.zeebe.exporter.ElasticsearchExporterException: Failed to flush all items of the bulk
	at io.zeebe.exporter.ElasticsearchClient.flush(ElasticsearchClient.java:153) ~[zeebe-elasticsearch-exporter-0.24.2.jar:0.24.2]
	at io.zeebe.exporter.ElasticsearchExporter.flush(ElasticsearchExporter.java:116) ~[zeebe-elasticsearch-exporter-0.24.2.jar:0.24.2]
	at io.zeebe.exporter.ElasticsearchExporter.flushAndReschedule(ElasticsearchExporter.java:103) ~[zeebe-elasticsearch-exporter-0.24.2.jar:0.24.2]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.24.2.jar:0.24.2]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.24.2.jar:0.24.2]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:118) [zeebe-util-0.24.2.jar:0.24.2]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.24.2.jar:0.24.2]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.24.2.jar:0.24.2]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:204) [zeebe-util-0.24.2.jar:0.24.2]

```

</p>
</details>

**Environment:**
 - Zeebe Version: 0.24.2
 - Configuration: set-up using helm chart
 - Elasticsearch version: 6.8.8

**Broker resource configuration**
 - CPUs: max - 2
 - Memory: max - 2 GiB

**Comment by npepinpe** [comment]
@saig0 - this sounds familiar. Is this something that would be fixed in new versions, or is this unrelated to previous issues we had with the mappings?

**Comment by npepinpe** [comment]
Hi @edward-sh, as this seems to be a problem with the Elasticsearch template, the best way for us to debug this is to know what data is being exported. There are two ways to do this:

1. Zip up the `/usr/local/zeebe/data` folder and attach it (via Google Docs or some other public place where we can grab it)
1. Share your workflows and sample payloads

This will help us figure out what's being exported so we can know why the Elastic template is rejecting it. Another option is to restart your broker with the following environment variable, `ZEEBE_DEBUG=true`, which will print out every single record. This adds considerable overhead however, and may print out sensitive information, so I will let you decide which is best for you. If you do enable it, then you can grab the logs and just attach them here.

**Comment by edward-sh** [comment]
Hello @npepinpe. I'm sorry for delay with answering. We have decided to update broker to the latest version at the next day after opening this issue and watch how it will work. Unfortunately we have purged all the data so it isn't possible to provide the information that you requested. We are monitoring our Zeebe Broker now and I think it's better to reopen this issue and provide all needed information to you if the problem will remain. Thank you for reply!
